nil/getting-started/essentials/reading-from-nil.mdx

```diff
@@ -0,0 +1,9 @@
+# Reading from =nil;
+
+This tutorial details the different ways in which information from =nil; could be read.
+
+## Via the CLI
+
+The =nil; CLI provides several commands that produce data about key cluster entities including blocks, messages, and logs. 
+
+To 
```

Comment:
There are now ways to read information via CLI. At least "wallet balance".

Also, you probably need to mention the block explorer (explore.nil.foundation)